packages/@o3r/stylelint-plugin/package.json

```
@@ -16,8 +16,8 @@
   "scripts": {
     "nx": "nx",
     "ng": "yarn nx",
-    "build": "yarn nx build stylelint-plugin",
-    "postbuild": "patch-package-json-main",
+    "build": "tsc -b tsconfig.build.json",
```

Generally we keep the `build` to run the nx task and do something like

```suggestion
    "build:source": "tsc -b tsconfig.build.json",
```